app/assets/stylesheets/layouts/_results.scss

```diff
@@ -19,6 +19,33 @@
   }
 }
 
+.l-results__firm-name {
+  @include column(12);
+
+  margin-left: 0;
```

Comment:
I think we can avoid manually collapsing margins if we wrap in something that uses the `row` mixin.